API.md

```diff
@@ -50,6 +46,9 @@ attached to `getBlock` are errored with `Error('manual unwant: key')`.
 
 Cancel previously requested keys.
 
+### `putStream()`
+
+Returns a duplex `pull-stream` that emits an object `{key: Multihash}` for every written block when it was stored.
 
 ### `hasBlock(block, cb)`
```

Comment:
`put` makes sense to me, yes it adds the block to the storage + tells the other bitswap parts that it has this block now. I copied the naming from the go implementation, that's where this comes from.